scripts/BaseL0Script.sol

```diff
@@ -63,10 +63,10 @@ contract BaseL0Script is L0Constants, Script {
     /// @dev alphabetical order as json is read in by keys alphabetically.
     struct NonEvmPeer {
         bytes32 fpi;
-        bytes32 frax;
+        bytes32 frxUSD;
         bytes32 frxEth;
         bytes32 fxs;
-        bytes32 sFrax;
+        bytes32 sfrxUSD;
```

Comment:
Note the difference between this line and the next of `sfrx` and `sFrx`.  Execution would be the same either way due to sorting, no change needed.